core/src/main/java/com/hcl/ecomm/core/filters/package-info.java

```diff
@@ -13,5 +13,7 @@
  *  See the License for the specific language governing permissions and
  *  limitations under the License.
  */
-// this file simply ensures that the resulting jar is not empty - see issue #30
-package com.hcl.ecomm.it.launcher;
+@Version("1.0")
```

Comment:
@DeepaliAgrwl : why do we have this java file ?

Reply:
removed the file